blocks/footer/footer.css

```diff
@@ -1,14 +1,312 @@
+  position: absolute;
+  left: 1.25rem;
+  top: 0;
+  bottom: 0;
   margin: auto;
 }
 
-footer .footer p {
-  margin: 0;
+.footer.block .footer-top .col-2::after {
+  height: 6rem;
+  background: var(--veritcal-line-footer-top-gray);
+  width: 0.0625rem;
+  content: "";
+  position: absolute;
+  right: 1.25rem;
+  top: 0;
+  bottom: 0;
+  margin: auto;
+}
+
+.footer.block .footer-disclaimer p {
+  color: var(--color-footer-gray);
+  font-size: 0.6875rem;
+  line-height: 0.8125rem;
+}
+
+.footer.block .footer-top .col-3 {
+  flex: 0 0 auto;
+  width: 25%;
+}
+
+.footer.block .footer-top .col-3 p {
+  text-align: left;
+  padding-left: 1rem;
+}
+
+.footer.block .footer-links {
+  background: var(--link-color-dark-gray);
+  padding: 6.875rem 0 0;
+} 
+
+.footer.block .footer-links .links-title {
+  font-size: var(--body-font-size-xs);
+  line-height: 130%;
+  color: var(--color-off-white);
+  font-weight: 500;
+  display: block;
+  margin-bottom: 0.75rem;
+  padding-top: 1rem;
+}
+
+@media (width > 48rem) {
+  .footer.block .footer-links .links-title {
+    font-size: var(--body-font-size-m);
+    line-height: 130%;
+    font-weight: 500;
+    margin-bottom: 0.75rem;
+  }
+}
+
+.footer.block .footer-links a {
+  color: var(--color-footer-gray);
+  transition: 0.3s;
+  line-height: 138%;
+  width: 100%;
+  max-width: 100%;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 12px;
+  padding: 0.3125rem 0;
+}
+
+@media (width > 48rem) {
+  .footer.block .footer-links a {
+    font-size: 0.8125rem;
+  }
+}
+
+.footer.block .footer-links a > u {
+  text-decoration: none;
+}
+
+.footer.block .footer-disclaimer a {
+  color: var(--link-color-orange);
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 0.6875rem;
+  transition: 0.3s;
+}
+
+.footer.block .footer-links a:hover {
+  color: var(--color-off-white);
+}
+
+.footer.block .footer-links .links-column {
+  flex: 0 0 auto;
+  width: 50%;
+}
+
+@media (width > 48rem) {
+  .footer.block .footer-links .links-column {
+    flex: 0 0 auto;
+    width: 16.6667%;
+    padding-left: 1rem;
+  }
+}
+
+.footer.block .footer-links .links-column > div {
+  margin: 0 0 1.25rem
+}
+
+@media (width < 37.5rem) {
+  .footer.block .footer-top-note p {
+    padding-left: 0.9375rem;
+    padding-right: 0.9375rem;
+  }
+}
+
+.footer.block .footer-top .row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -0.9rem;
+  margin-left: -0.9rem;
+}
+
+.footer.block .footer-links .row {
+  --bs-gutter-x: 1.5rem;
+  --bs-gutter-y: 0;
+
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: calc(-1* var(--bs-gutter-y));
+  margin-right: calc(-.5* var(--bs-gutter-x));
+  margin-left: calc(-.5* var(--bs-gutter-x));
+}
+
+.footer.block .footer-links .row>* {
+  padding-right: calc(var(--bs-gutter-x)* .5);
+  padding-left: calc(var(--bs-gutter-x)* .5);
+  margin-top: var(--bs-gutter-y);
+}
+
+@media (width >= 56.25rem) {
+  .footer.block .section > .section-container {
+    max-width: 81.25rem;
```

Comment:
Screensize 56.25rem and max-width 81.25rem may be there needs to be another media query break point. just check once

Reply:
fixed. if >=81.25 then that would be max-width